CONTRIBUTING.md

```diff
@@ -56,37 +55,188 @@ Please include the following with each issue:
+- `imodel-components-react/` contains components that require a 3D environment. Used exclusively for iModels.
+
+### Common issues to solve
+
+Common fixes are we looking to solve include, but ar e not limited to:
+
+- Replacing `core-react` components with iTwinUI components
+- Deleting Snapshot tests
+- Migrating to functional from class based React Components
+
+### Conventions
+
+Please follow these conventions when contributing to AppUI
+
+- Be sure that your branch is up to date with the master branch (i.e. `git merge origin/master`)
+- Classnames should follow this formula: [abrv. of module name] + [folder] + [component]. For example, when working inside the Message Center module on MessageCenterField on the dialog, one would use `”mc-messagecenterfield-dialog”`
+- All components should import a single SCSS file with a matching name, e.g. `Message.tsx` importing `Message.scss`. SCSS files should be siblings of their relative component files.
+
+---
+
+## Contributor License Agreement (CLA)
 
 A [Contribution License Agreement with Bentley](https://gist.github.com/imodeljs-admin/9a071844d3a8d420092b5cf360e978ca) must be signed before your contributions will be accepted. Upon opening a pull request, you will be prompted to use [cla-assistant](https://cla-assistant.io/) for a one-time acceptance applicable for all Bentley projects.
 You can read more about [Contributor License Agreements](https://en.wikipedia.org/wiki/Contributor_License_Agreement) on Wikipedia.
 
-### Pull Requests
+---
 
-All submissions go through a review process.
-We use GitHub pull requests for this purpose.
-Consult [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more information on using pull requests.
+## Testing options
 
-### Types of Contributions
+The repository is set up to allow 2 different ways of testing changes with manual interactions.
 
-We welcome contributions, large or small, including:
+### Test apps
 
-- Bug fixes
-- New features
-- Documentation corrections or additions
-- Example code snippets
-- Sample data
+In the `test-apps` folder there are 2 apps that can be used to test changes to the packages in this repository. Each app is a standalone app that can be run with `npm start` from the app's folder.
 
-Thank you for taking the time to contribute to open source and making great projects like iTwin.js possible!
+Most of the features should be the same in both apps as they are both being configured by the `appui-test-providers` package, new features should be added through this package.
+
+The apps are:
+
+- `standalone`: This app is working only on the current machine and do not require log in, it is useful for testing with `.bim` files that you are on your machine. [See Readme for more info](./test-apps/appui-test-app/standalone/README.md)
```

Comment:
```suggestion
- `standalone`: is an iTwin.js application that is using IPC and is useful for testing with local `.bim` files that are on your machine. [See Readme for more info](./test-apps/appui-test-app/standalone/README.md)
```

Reply:
If we go with this sentence structure, the semicolon should probably be removed as well. Also I believe it sounds better if we write `application that uses IPC` instead of `that is using`.